blobfuse2-code-coverage.yaml

```diff
@@ -374,6 +374,28 @@ jobs:
         workingDirectory: $(WORK_DIR)
         displayName: "CLI : Config change simulator"
 
+      # Secure Config, fine to use insecure passphrase as this is just for testing
+      - script: |
+          rm -rf $(MOUNT_DIR)/*
+          rm -rf $(TEMP_DIR)/*
+          ./blobfuse2 unmount all
+          
+          ./blobfuse2.test -test.v -test.coverprofile=$(WORK_DIR)/mount_secure.cov secure encrypt --config-file=$(BLOBFUSE2_CFG) --output-file=$(Pipeline.Workspace)/blobfuse2.azsec --passphrase=123123123123123123123123 
+          if [ $? -ne 0 ]; then
+            exit 1
+          fi
+          ./blobfuse2.test -test.v -test.coverprofile=$(WORK_DIR)/mount_secure.cov mount $(MOUNT_DIR) --config-file=$(Pipeline.Workspace)/blobfuse2.azsec --passphrase=123123123123123123123123 &
```

Comment:
Do not use the same coverage file name, otherwise data from last test will be lost. "/mount_secure.cov"

Reply:
oh i see. will fix